README.md

```diff
@@ -8,3 +8,15 @@
 A simple calculator application built using the Fyne toolkit.
 
 ![](img/calc-dark.png) &nbsp; ![](img/calc-light.png)
+
+# Installation instructions
+Install Fyne:
+```go get fyne.io/fyne/v2/cmd/fyne```
+
+Install resources for this application:
+go to folder /calculator
+and execute: 
+```fyne install```
```

Comment:
I think it might look better if it was like this instead (with code blocks instead of just formatting):
```
fyne install
```
You just need to put the three dots on a new line before and after the “fyne install” text.

It applies to more places.